docs/simulate/pauli_observables.ipynb

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        typrint((3j * Xa * Zb) ** 3)
</pre><p>maybe also note that this works with <code>math.exp</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5750/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>It doesn't seem to...</p>

Reply:
<p>Oh, interesting.  Seems numpy falls back to <code>pow</code> but math does not.   </p>